addons/base_import_module/models/ir_module.py

```diff
@@ -194,6 +194,16 @@ def _import_module(self, module, path, force=False, with_demo=False):
             'res_id': asset.id,
         } for asset in created_assets])
 
+        translation_importer = TranslationImporter(self.env.cr, verbose=False)
+        for lang_ in self.env['res.lang'].get_installed():
+            lang = lang_[0]
+            po_paths = get_po_paths(module, lang, env=self.env)
+            for po_path in po_paths:
+                translation_importer.load_file(po_path, lang)
+            if lang != 'en_US' and not po_paths:
```

Comment:
after odoo/odoo#135607
the `po_paths` is a generator, so `bool(po_paths)` is always `True`
here you cannot check `no translation for language`
maybe also fix it in `def _load_module_terms`